mysql-test/suite/rocksdb/t/index_merge_rocksdb-master.opt

```diff
@@ -0,0 +1 @@
+--loose-rocksdb_strict_collation_check=off --binlog_format=row --log-bin
```

Comment:
Dunno, came from upstream and I didn't really notice. Good question as binlog/rpl shouldn't have anything to do with this test.